apps/dav/lib/BackgroundJob/UploadCleanup.php

```diff
@@ -66,6 +67,9 @@ protected function run($argument) {
 		} catch (NotFoundException $e) {
```

Comment:
```suggestion
		} catch (NotFoundException|NoUserException $e) {
```

Does the same, without the duplication :)

Reply:
Ah right, fixed :+1: